extensions/amp-social-share/amp-social-share.md

```diff
@@ -246,43 +262,15 @@ Sometimes you want to provide your own style. You can simply override the provid
 
 ```css
 amp-social-share[type='twitter'] {
+  color: blue;
   background: red;
-  background-image: url(datauri:svg/myownsvgicon);
 }
 ```
 
 When customizing the style of an `amp-social-share` icon please ensure that the customized icon meets the branding guidelines set out by the provider (e.g Twitter, Facebook, etc.)
 
 ## Accessibility
 
-### Indication of focus
-
-The `amp-social-share` element defaults to a blue outline as a visible focus indicator. It also defaults `tabindex=0` making it easy for a user to follow along as he or she tabs through multiple `amp-social-share` elements used together on a page.
-
-The default focus indicator is achieved with the following CSS rule-set.
-
-```css
-amp-social-share:focus {
```

Comment:
This is still possible and potentially relevant, no?

Reply:
Added to tracker issue here: https://github.com/ampproject/amphtml/issues/28283